kotlin/internal/jvm/compile.bzl

```diff
@@ -325,7 +325,9 @@ def _run_kt_builder_action(ctx, rule_kind, toolchains, dirs, srcs, friend, compi
 
     ctx.actions.run(
         mnemonic = "KotlinCompile",
-        inputs = depset(ctx.files.srcs, transitive = [compile_deps.compile_jars]),
+        inputs = depset(
+            ctx.files.srcs,
+            transitive = [compile_deps.compile_jars] + [ap.classpath for ap in annotation_processors.to_list()]),
```

Comment:
I'm not sure if the `to_list()` is strictly required, but I'm not sure how this would handle a depset of structs rather than a depset of files.

Reply:
We want to avoid to_list like the plague, unfortunately. It would be better to export the processor classpath from https://github.com/bazelbuild/rules_kotlin/blob/master/kotlin/internal/jvm/plugins.bzl#L36 to limit the number of times we call to_list. (probably should even happen there, tbh.

Reply:
I think I've made the change you requested. I couldn't do it within the exact method you pointed to, not that I could figure anyways, but I've made changes similar to the existing pattern.